test/other/test_emsize.out

```diff
@@ -1,11 +1,13 @@
 section      size   addr
-TYPE           51     10
-IMPORT         31     63
-FUNCTION       14     96
-GLOBAL          9    112
-EXPORT          9    123
-ELEM            9    134
-CODE         1556    146
-DATA           77   1704
-JS		6756	0
-Total		8512
+TYPE           41     10
+IMPORT          7     53
+FUNCTION        9     62
+TABLE           5     73
+MEMORY          6     80
+GLOBAL          8     88
+EXPORT         17     98
+ELEM            9    117
+CODE         1083    129
+DATA           72   1214
+JS		4799	0
+Total		6056
```

Comment:
Its nice to see the progress we've made on overall code size here since this test was last rebaselined